src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/LocalDBAPI.cs

```diff
@@ -32,7 +32,10 @@ internal static string GetLocalDbInstanceNameFromServerName(string serverName)
         {
             if (serverName == null)
                 return null;
-            serverName = serverName.TrimStart(); // it can start with spaces if specified in quotes
+            serverName = serverName.TrimStart(null); // it can start with spaces if specified in quotes
```

Comment:
Since the code pattern and functionality is the same I did it for consistency sake in both framework. I wait for other reviewers to vote before taking that out.

Reply:
Using the same overload in all builds would be preferable to me. It would be nice to have a comment identifying why the null parameter is used so someone doesn't optimize it back in future.